.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Issue Title
+
+Be as descriptive as you can with your title. If your issue is requesting
+a new topic, please preface the title with [NewTopic].
+
+# General
+
+We use issues to drive the discussion for changes to
+existing topics, and the creation of new topics.
+
+This way, we have those discussions before significant
```

Comment:
That looks more like a product issue template. I think ours should be a bit different.